src/components/BookList.jsx

```diff
@@ -0,0 +1,69 @@
+import { useDispatch } from 'react-redux';
+import React from 'react';
+import PropTypes from 'prop-types';
+import { deleteBook } from '../redux/slices/books/bookSlice';
+
+function BookList({ books }) {
+  const dispatch = useDispatch();
+
+  return (
+    <ul>
+      {books.map((book) => (
+        <>
+          <li
+            key={book.item_id}
+            style={{
+              display: 'flex',
+              alignItems: 'center',
+              justifyContent: 'space-between',
+              marginTop: '50px',
+            }}
+          >
+            <div>
+              <span>{book.category}</span>
+              <h2>{book.title}</h2>
+              <p>{book.author}</p>
+              <div style={{
+                display: 'flex',
+                gap: '10px',
+                marginTop: '25px',
+              }}
+              >
+                <button type="submit">Comments</button>
+                <button onClick={() => dispatch(deleteBook(book.item_id))} type="submit">Remove</button>
+                <button type="submit">Edit</button>
+              </div>
+            </div>
+            <div style={{
```

Comment:
- [ ] To make your codebase clean and your components reusable, i will recommend that you create a separate component called <Book /> which will contain a single book item. You can then import it into your <BookList /> component where you will then map to display all your books. This way, your code will be modular and readable.